.github/workflows/close.yaml

```diff
@@ -0,0 +1,34 @@
+# This workflow warns and then closes issues and PRs that have had no activity for a specified amount of time.
+#
+# You can adjust the behavior by modifying this file.
+# For more information, see:
+# https://github.com/actions/stale
+name: Close Old
+
+on:
+  workflow_dispatch:
+  schedule:
+  - cron: '0 1 1,15 * *'
+
+jobs:
+  stale:
+
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+      
+
+    steps:
+    - uses: actions/stale@v5
+      with:
+        repo-token: ${{ secrets.GITHUB_TOKEN }}
+        days-before-issue-stale: 30
+        stale-issue-message: >
+          @author, We are closing this as there is no activity in this issue for last 90 days. Please reopen if you’d like to discuss anything further.
```

Comment:
I think the better way to do this is to use `close-issue-message` and `days-before=close` on the obsolete workflow, since that is the workflow I see in the github action docs:

e.g. 
https://github.com/actions/stale/tree/main#close-issue-message:~:text=Configure%20different%20stale%20timeouts%3A